monsieurkayode/More-Recipes-Kayode-Adeola
# #152667219 Refactor Component Layout

#### What does this PR do?
Make changes to component composition for headers
#### Description of Task to be completed?
* Connect the Navigation Bar components to the store
* Fix issue with Logout not working in some views
#### How should this be manually tested?
* Clone the repo and install all dependencies using `npm install` && `cd client && npm install` for server and client app dependencies
* Setup your database, cd int the `config` folder in server and rename to `database` under development in config.json to your database name
* Run `npm migrate`
* Start both the server and client by running `npm start` and `cd client && npm start`
* Add a `.env` file, see readme for sample
* Create an account and login
* Logout using the dropdown button at the top left corner of the header
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152667219
